src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -1,12 +1,13 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
 /**
  * Parameterize this service and add its implementation in a separate class.
  */
-public interface MachineService {
 
+public interface MachineService<T> {
```

Comment:
Don't begin class or method implementation with empty lines. Please remove the excess empty lines to improve code readability.